doc/nrf/ug_gzll.rst

```diff
@@ -52,8 +45,8 @@ To enable the Gazell support in the |NCS|, set the following Kconfig options:
 
 .. _ug_gzll_resources:
 
-Resources
-*********
+Resources required
+******************
 
 Gazell uses a fixed set of peripheral resources in the nRF5 SoC.
```

Comment:
According to this page, `nRF5` is not allowed as a term in out docs: https://projecttools.nordicsemi.no/confluence/display/MAR/Product+names

Reply:
Okay, will fix this. Will ask about this first.

Reply:
E.g., ...in the SoCs of the nRF52 Series (confirm this)